libraries/AP_Avoidance/AP_Avoidance.cpp

```diff
@@ -418,8 +415,7 @@ void AP_Avoidance::handle_threat_gcs_notify(AP_Avoidance::Obstacle *threat)
         // only send cleared messages for a few seconds:
         if (_gcs_cleared_messages_first_sent == 0) {
             _gcs_cleared_messages_first_sent = now;
-        }
-        if (now - _gcs_cleared_messages_first_sent > _gcs_cleared_messages_duration * 1000) {
+        } else if (now - _gcs_cleared_messages_first_sent > _gcs_cleared_messages_duration * 1000) {
```

Comment:
This is good but should have been in its own commit

Reply:
agreed. @muramura the secret to getting PRs merged quickly is to make sure they are "obviously correct". Multiple PRs are better than fix-creep like this